### PR TITLE
 creation page profil

### PR DESCRIPTION
**Pull request by n-faycel**

### CODE REVIEW

**Review by Dash293 — APPROVED** [review]

File: frontend/src/components/UserProfil/style.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    font: var(--fontTitleBold);
+  }
+
+  .contenairname {
+    display: flex;
+    justify-content: flex-start;
+    height: 17vh;
+    width: 85vw;
+    margin: auto;
+
```
Comment (LuigiMenez) [inline frontend/src/components/UserProfil/style.jsx:28]: Blank line

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by Dash293 — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Okk for me.

**Review by LoicBrassart — APPROVED** [review]
SnE (except, you know, you don't folow instructions and pump more code everytime I ask for a little fix :smirk: )

**Comment by LoicBrassart** [comment]
Sorry about my last comment, GHub made me reread whole files instead of just showing what you had modified.
My bad ! :pray:

**Pull request merged**